source/tests/Test.cpp

```diff
@@ -29,10 +29,34 @@ Test::Test() : global_redex_context_(/* allow_class_duplicates */ false){};
 ContextGuard::ContextGuard()
     : global_redex_context_(/* allow_class_duplicates */ false){};
 
+/**
+ * Empty options. Useful to initialize an empty context.
+ */
+std::unique_ptr<Options> make_empty_options() {
+  return std::make_unique<Options>(
+      std::vector<std::string>(),
+      std::vector<std::string>(),
+      std::vector<std::string>(),
+      std::vector<std::string>(),
+      std::vector<std::string>(),
+      std::vector<std::string>(),
+      ".",
+      std::vector<std::string>(),
+      true,
+      true,
+      true,
+      std::vector<ModelGeneratorConfiguration>(),
+      std::vector<std::string>(),
+      true,
+      true
+    );
+}
+
```

Comment:
style: we typically prefix each argument with a comment on what it is just for readability
```
  return std::make_unique<Options>(
      /* models_path */ std::vector<std::string>(),
      /* field_models_path */ std::vector<std::string>(),
...
    );
```

Reply:
Solved in 755c20b93f31e09fe561ca491a28c2225b621572